server/router/signin.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+
```

Comment:
Not sure if we need both signin and login routers? I think we just need one for signing in to your Oauth?